config/sidebar.paper.ts

```diff
@@ -3,174 +3,177 @@ import type { SidebarsConfig } from "@docusaurus/plugin-content-docs";
 const paper: SidebarsConfig = {
   primary: [
     "README",
-    {
-      type: "category",
-      label: "Administration",
-      collapsed: true,
-      link: {
-        type: "doc",
-        id: "admin/README",
+    "admin/README",
+    "dev/README",
+    "contributing/README",
+  ],
+  administration: [{
+    type: "category",
+    label: "Administration",
+    collapsed: true,
+    link: {
+      type: "doc",
+      id: "admin/README",
+    },
+    items: [
+      {
+        type: "category",
+        label: "Getting Started",
+        link: {
+          type: "generated-index",
+          slug: "/cat/admin/getting-started",
```

Comment:
Why is `cat` here? The development docs don't have it

Reply:
Cause the docs have horribly enforced standards :P